src/playground/ec/.storybook/preview.js

```diff
@@ -45,6 +45,12 @@ export const parameters = {
       picked: true,
       hideCode: true,
     },
+    {
```

Comment:
i don't know, in the end this is going to be a pretty small file, not sure why keeping it separated, maybe because there might be sites not using it at all..? I mean, do we have any advantage by doing this? What i initially thought was that we could have produced one file per color mode, so that the single website would have been only loading the one corresponding to its chosen "priority", but if we are about to ship them all together then i don't necessarily see why keeping it separated, it will be combined by the application in any case..

Reply:
The main reason why I propose to split it is because of possible request to add/update color mode quickly, if there is some business need.
Having it in its own little css would allow to update the colors first, and take the time to really catch up with the ECL release. I agree that it is an edge case (and somehow hacky), so maybe we should see first if that may be a real case or not